Merge MI-108 with dev

AlineTheAlien opened the pull request:
Changed setGSD method to only accept a dynamic object from body

AlineTheAlien committed:
#65 Changed setGSD method in UserController to accept an object instead of an int

nim442 committed:
Merge branch 'dev'

AlineTheAlien commented:
@nim442 Can you please review?

nim442 commented on src/mRides-server/Controllers/UserController.cs:
```diff
@@ -98,10 +98,11 @@ public long getGSD(int id)
+        public long setGSD([FromBody]dynamic sentObject)
         {
-            int newUserId = Convert.ToInt32(id);
+            //int newUserId = Convert.ToInt32(userId);
```
remove commented out code

AlineTheAlien replied:
Good catch!

nim442 commented on src/mRides-server/Logic/UserCatalog.cs:
```diff
@@ -144,15 +144,13 @@ public User create(User obj, int userId)
 
         public void setGSD(int userId, long amountGSD)
         {
-            //User u = _context.Users.Find(userId);
```
👍

AlineTheAlien committed:
Removed uncessesary inline comments from UserController class.

These comments are redundant and confusing as they are past code therefore they are removed to keep the code more concise.

AlineTheAlien commented:
@nim442 I made the changes you requested, the branch is ready to merge.
